backend/cfg/cfg_polling.ml

```diff
@@ -127,8 +127,8 @@ let is_safe_terminator : Cfg.terminator Cfg.instruction -> bool =
   | Always _ | Parity_test _ | Truth_test _ | Float_test _ | Int_test _
   | Switch _ ->
     false
-  | Raise _ -> false
-  | Tailcall_self _ | Tailcall_func _ | Return -> true
+  | Raise _ | Tailcall_self _ -> false
+  | Tailcall_func _ | Return -> true
```

Comment:
`Tailcall_func` should also be considered not safe if we remove the prologue_poll entirely.